packages/safe-ds-eda/src/webviewState.ts

```diff
@@ -26,6 +26,32 @@ window.addEventListener('message', (event) => {
             // This should be fired immediately whenever the panel is created or made visible again
             currentState.set(message.value);
             break;
+        case 'setProfiling':
+            if (get(currentState) && get(currentState).table) {
+                currentState.update((state) => {
+                    return {
+                        ...state,
+                        table: {
+                            ...state.table!,
+                            columns: state.table!.columns.map((column) => {
+                                const profiling = message.value.find((p) => p.columnName === column[1].name);
+                                if (profiling) {
+                                    return [
+                                        column[0],
```

Comment:
Some thoughts/questions:

1. `column[0]`/`column[1]` reads like getting the first/second value of a column. However, IIUC, `column[0]` is some index and `column[1]` is the actual column. This naming is very confusing.
2. What is the purpose of `column[0]`? It seems to only ever be written but never read. Moreover, the actual column has a name, which we can use as an identifier.

Reply:
column[0] is for the index of order of the column, so what position it is, initially planned for restoring it after fetching a new table from the Runner through an action. When faced with the action history and restoring it (in the next issue) I was looking a lot more at this and will change how it works, so will be addressed there.

Reply:
How about storing the initial index in the column object (i.e. column[1]) then?

Reply:
My plan was to have all the internal actions (not requiring a runner call, like hide column, reorder etc.) be restored inside the webview.  Since internal actions will be stored in the history too, they may not need extra saving in the state so I could restore the column order from just the history and thus get rid of the index in the state object for columns. Then in the history object for reordering I will store the new order as a map of column name to index and whenever the table updates with a request to the Runner I will iterate over the table to put it in the right order of the last reorder history item before showing.